include <stdexcept> in polynomial.cc

I think <stdexcept> needs to be included in polynomial.cc

I'm building madness with gcc 10.2.0 on an ubuntu 16.04 system.
I get the following error when compiling src/apps/chem/polynomial.cc
polynomial.cc:121: error: 'runtime_error' is not a member of 'std'

If I include <stdecept> in polynomial.cc, this error does not appear, and the rest of my build completes

--- a/src/apps/chem/polynomial.cc
+++ b/src/apps/chem/polynomial.cc
@@ -9,6 +9,7 @@
  */
 
 #include <cmath>
+#include <stdexcept>
 #include <vector>
 #include "polynomial.h"
 

